playbooks/init/base_packages.yml

```diff
@@ -35,7 +35,6 @@
       - >
         (openshift_use_system_containers | default(False)) | bool
         or (openshift_use_etcd_system_container | default(False)) | bool
```

Comment:
do we need to nuke `openshift_use_etcd_system_container` everywhere?

Reply:
We've got a card to rip all of the system containers code from components we don't intend to support going forward. https://trello.com/c/8NYDYrnL/604-5-remove-all-other-tech-preview-system-containers-except-kubelet-docker-cri-o-from-the-product